src/components/screens/SendScreen/components/RecentTransactionsList.tsx

```diff
@@ -0,0 +1,82 @@
+import { ContactRow } from "components/screens/SendScreen/components";
+import Icon from "components/sds/Icon";
+import { Text } from "components/sds/Typography";
+import useAppTranslation from "hooks/useAppTranslation";
+import useColors from "hooks/useColors";
+import React from "react";
+import { FlatList, View, KeyboardAvoidingView, Platform } from "react-native";
+
+interface RecentTransaction {
+  id: string;
+  address: string;
+  name?: string;
+}
+
+interface RecentTransactionsListProps {
+  transactions: RecentTransaction[];
+  onContactPress: (address: string) => void;
+  testID?: string;
+}
+
+/**
+ * Header component for the recent transactions list
+ *
+ * @returns {JSX.Element} The rendered header component
+ */
+const ListHeader = () => {
+  const { t } = useAppTranslation();
+  const { themeColors } = useColors();
+
+  return (
+    <View className="mb-[24px]">
+      <View className="flex-row items-center gap-2">
+        <Icon.Clock size={16} color={themeColors.foreground.primary} />
+        <Text md medium secondary>
+          {t("sendPaymentScreen.recents")}
+        </Text>
+      </View>
+    </View>
+  );
+};
+
+/**
+ * Displays a list of recent transactions/contacts
+ *
+ * @param {RecentTransactionsListProps} props - Component props
+ * @returns {JSX.Element | null} The rendered component or null if no transactions
+ */
+export const RecentTransactionsList: React.FC<RecentTransactionsListProps> = ({
```

Comment:
I'd suggest renaming it to `RecentAddressesList` or `RecentContactsList` for extra clarity since it's showing address/contact details instead of the actual transaction details

Reply:
I'd vote for `RecentContactsList` since we're using `ContactRow` for the list items and